transaction-context/src/lib.rs

```diff
@@ -725,50 +714,43 @@ impl InstructionContext {
             })
     }
 
-    fn try_borrow_account<'a, 'b: 'a>(
+    /// Get the owner of the program account of this instruction
+    pub fn get_program_owner(
+        &self,
+        transaction_context: &TransactionContext,
+    ) -> Result<Pubkey, InstructionError> {
+        self.get_index_of_program_account_in_transaction()
+            .and_then(|index_in_transaction| {
+                transaction_context
+                    .accounts
+                    .try_borrow(index_in_transaction)
+            })
+            .map(|acc| *acc.owner())
```

Comment:
Do we have to always clone the Pubkey returned here? Can't we make it a ref with a lifetime of the `transaction_context` parameter?